tap-setup-scripts/src/inputs/user-input-values.example.yaml

```diff
@@ -13,7 +13,7 @@ tap:
   name: tap
   namespace: tap-install
   repository: registry.tanzu.vmware.com/tanzu-application-platform/tap-packages
-  version: 1.2.0
+  version: 1.3.0-rc.1
```

Comment:
this is just a sample example file. The values are for illustration purposes only. not actually used.